dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionControllerTest.java

```diff
@@ -195,8 +195,8 @@ public void testQueryProcessDefinitionByCode() {
         putMsg(result, Status.SUCCESS);
         result.put(Constants.DATA_LIST, processDefinition);
 
-        Mockito.when(processDefinitionService.queryProcessDefinitionByCode(user, projectCode, code)).thenReturn(result);
-        Result response = processDefinitionController.queryProcessDefinitionByCode(user, projectCode, code);
+        Mockito.when(processDefinitionService.queryProcessDefinitionByCode(user, projectCode, code, false)).thenReturn(result);
```

Comment:
can you add the test case of `showSubtask = true`?

Reply:
I met a problem in the ut, I need to query the sub-process in the database, do you know how I should generate this part of the test data.